compare.go

```diff
@@ -440,9 +427,45 @@ func Compare(oldDh, newDh *DirectoryHierarchy, keys []Keyword) ([]InodeDelta, er
 					new:  *diff.New,
 					keys: changed,
 				})
+			} else if same {
+				results = append(results, InodeDelta{
```

Comment:
would be nice to see a slight comment informing how the remainder of this case is that all the files are same. Redundant, but an explicit fall through.

Reply:
Sure, I added a small comment.